### PR TITLE
Compiler-independent struct align macro

### PR DESCRIPTION
**Pull request by VelocityRa**
Improves compilation status on MSVC.

### CODE REVIEW

File: include/vitasdk/align.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#ifdef _MSC_VER
+#    if (_MSC_VER >= 1800)
+#        define __alignas_is_defined 1
+#    endif
+#    if (_MSC_VER >= 1900)
+#        define __alignof_is_defined 1
+#    endif
+#else
+#    include <cstdalign>   // __alignas/of_is_defined directly from the implementation
```
Comment (d3m3vilurr) [inline include/vitasdk/align.h]: is it allow in gcc?

**Comment by d3m3vilurr** [comment]
you should update `.travis.d/definition_check.py` to pass the travis teesting. :)

**Comment by VelocityRa** [comment]
@d3m3vilurr I added it here https://github.com/vitasdk/vita-headers/pull/438/files#diff-1bf3e94b741b213928567cb72bdea7c0R30 what else should I do?

File: include/vitasdk/align.h (hunk 1 above)
Comment (VelocityRa) [reply include/vitasdk/align.h]: Apparently your gcc version doesn't support it? Not sure, I'll remove that.

**Comment by VelocityRa** [comment]
Ok builds now, used another method.

File: include/vitasdk/align.h (hunk 1 above)
Comment (d3m3vilurr) [reply include/vitasdk/align.h]: cstdalign is c++ header. so i asked it. tha t would be only allowed with c++ flags :)

_[FastHub-Libre](https://github.com/thermatk/FastHub-Libre/)을 사용하여 Mi 5에서 보냈습니다_

Comment (VelocityRa) [reply include/vitasdk/align.h]: Ah right.

File: include/vitasdk/align.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#ifndef _VITASDKALIGN_H_
+#define _VITASDKALIGN_H_
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+#if defined(_MSC_VER)
+#define ALIGN(x) __declspec(align(x))
```
Comment (d3m3vilurr) [inline include/vitasdk/align.h]: could you give another name like `SCE_ALIGN` or `PSP2_ALIGN`?

**Pull request merged**